diff --git a/android/src/main/java/expo/modules/updates/launcher/DatabaseLauncher.kt b/android/src/main/java/expo/modules/updates/launcher/DatabaseLauncher.kt
index 1f7610f384d81ea0f32ee7600a0942eceecb16b3..692bebe02c404645ccf6a41b833eafee4c2eec24 100644
--- a/android/src/main/java/expo/modules/updates/launcher/DatabaseLauncher.kt
+++ b/android/src/main/java/expo/modules/updates/launcher/DatabaseLauncher.kt
@@ -150,6 +150,11 @@ class DatabaseLauncher(
     val embeddedUpdate = EmbeddedManifestUtils.getOriginalEmbeddedUpdate(context, configuration)
     val filteredLaunchableUpdates = mutableListOf<UpdateEntity>()
     for (update in launchableUpdates) {
+      var status = update.status.name
+      var id = embeddedUpdate?.updateEntity?.id?.toString()
+      var other = update.id.toString()
+      var config = configuration.hasEmbeddedUpdate.toString()
+      logger.info("UPDATE IS update.status=$status embeddedUpdate.updateEntity.id=$id update.id=$other configuration.hasEmbededUpdate=$config")
       // We can only run an update marked as embedded if it's actually the update embedded in the
       // current binary. We might have an older update from a previous binary still listed as
       // "EMBEDDED" in the database so we need to do this check.
